nats.go

```diff
@@ -2424,6 +2424,47 @@ func (ac *asyncCallbacksHandler) asyncCBDispatcher() {
 	}
 }
 
+// Returns the JS account info containing applicable limits
+// If JetStream is not enabled, this will return ErrJetStreamNotEnabled
+// Other errors can happen but are generally considered retryable
+func (nc *Conn) jsAccountInfo(jOpts *jsOpts, opts ...JSOpt) (*AccountInfo, error) {
+	o, cancel, err := getJSContextOpts(jOpts, opts...)
+	if err != nil {
+		return nil, err
+	}
+	if cancel != nil {
+		defer cancel()
+	}
+
+	resp, err := nc.RequestWithContext(o.ctx, jOpts.apiSubj(apiAccountInfo), nil)
```

Comment:
If user provides a different prefix than the one registered when initially getting the context, then should not this prefix be used instead? In other words, should it be `o.apiSubj()` here? (if I understand correctly `o` is the mix of the default's js options and the user provided one.

Reply:
fixed. instead of moving code we now use js.AcccountInfo (that existed before)